### DISCUSSION TITLE
Cannot get MDI to work with the serial terminal

### DISCUSSION THREAD

**Discussion by bohbotjames** [vlachoudis/bCNC#1873] [General]
Hi,

I have bCNC setup and the machine moves on all axis. I have calibrated the steps/mm too. Now I want to use the serial terminal to send manual commands to the board, but there is nothing happening on the terminal. Only the box to enter commands type fine, but when I send it, the command disappears as it should, but the terminal does not display anything, and the machine doesn't react. 

I am sure there is user error or something I am not doing to make it work, I just don't know what it is. Help is appreciated, thank you.

**Comment by MARIOBASZ** [comment]
Maybe by switching to the terminal tab you can see what's happening?

**Comment by MARIOBASZ** [comment]
Write $$? or G0 x 10?
Something that has happened to me: maybe you wrote g1 without setting speed

**Comment by bohbotjames** [comment]
Hey @MARIOBASZ , I tried typing $$ in the command window below the connecting button on the File tab. I cannot seem to get it to output status data of the board connected to bCNC

As I said, my Arduino serial terminal works, and I set $10 to 3 on there, and all the controls work on bCNC, just not the terminal. I need the terminal to home my machine with the $H command. bCNC doesn't seem to have a home button...

**Comment by MARIOBASZ** [comment]
In the Control tab, the first button (says home) is for homing

**Reply by bohbotjames** [reply]
Thank you <3
